common/src/main/resources/deployments/kafka/strimzi.yaml

```diff
@@ -47,30 +47,17 @@ items:
            terminationGracePeriodSeconds: 10
            containers:
              - name: kafka-broker-1
-               image: quay.io/debezium/kafka
+               image: strimzi/kafka:0.18.0-kafka-2.5.0
```

Comment:
- Why this switch?
- There is newer version available - `0.20.1-kafka-2.5.0` https://hub.docker.com/r/strimzi/kafka/tags
- There is also https://hub.docker.com/r/strimzi/kafka/tags with `0.21.1-kafka-2.5.0` and `0.21.1-kafka-2.7.0`

Reply:
I tried with a newer version but it failed at container startup (something about permissions). 
That's why I created this ticket https://trello.com/c/eZWV77rM/582-use-strimzi-kafka-version-from-quayio

Reply:
Both strimzi and debezium. are RH driven, but agree strimzi should be prioritized.